packages/extension/src/ui/features/accountTokens/tokens.service.ts

```diff
@@ -23,20 +23,22 @@ const formatTokenBalanceToCharLength =
     (balance: BigNumberish = 0, decimals = 18): string => {
       const balanceBn = BigNumber.from(balance)
       const balanceFullString = utils.formatUnits(balanceBn, decimals)
-
       // show max ${length} characters or what's needed to show everything before the decimal point
       const balanceString = balanceFullString.slice(
         0,
         Math.max(length, balanceFullString.indexOf(".")),
```

Comment:
Question: If I have a token, the `decimals` is 0, and the `balance` is `1111111111111`, which has a length greater than the `length` (currently 9), will this also cause the formatted balance to be incorrect?

Reply:
Good catch, in that case we should show the full string :+1: